Anlab.Mvc/Views/Alerts/Edit.cshtml

```diff
@@ -0,0 +1,57 @@
+@model SystemAlert
+
+@{
+    ViewBag.Title = "Edit Alert";
+}
+
+<div class="col">
+
+    <form asp-action="Edit">
+        <div class="form-horizontal">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+
+            <div class="form-group">
+                <label asp-for="Description" class="col-md-2 control-label"></label>
+                <div class="col-md-10">
+                    <input asp-for="Description" class="form-control" />
+                    <span asp-validation-for="Description" class="text-danger"></span>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label asp-for="Markdown" class="col-md-2 control-label"></label>
+                <div class="col-md-10">
+                    <textarea asp-for="Markdown" class="form-control" rows="10"> </textarea>
+                    <span asp-validation-for="Markdown" class="text-danger"></span>
+                    <div>
+                        <small>To see basic commands on how markdown works, google Markdown cheat sheet. Not all options are supported with the markdown library we use but most are. You may also add in html, but we recommend that you don't.</small>

```

Comment:
will html work?  i would think the model binder would auto reject html as a potential security issue

Reply:
Yes, it just goes in as raw. Same thing the other place we use this.
I only tested with <br/> but it is expected behavior with markdown from what I can see.